### PR TITLE
[Form] add how to get wrappedComponent's ref

### PR DESCRIPTION
**Pull request by muzea**
relate #10960

### CODE REVIEW

File: components/form/index.en-US.md [hunk 1]
```diff
@@ -57,6 +57,23 @@ The following `options` are available:
 | validateMessages | Default validate message. And its format is similar with [newMessages](https://github.com/yiminghe/async-validator/blob/master/src/messages.js)'s returned value | Object { [nested.path]&#x3A; String } |
 | onFieldsChange | Specify a function that will be called when the value a `Form.Item` gets changed. Usage example: saving the field's value to Redux store. | Function(props, fields) |
 | onValuesChange | A handler while value of any field is changed | (props, changedValues, allValues) => void |
+| withRef | Whether `refs` contain `wrappedComponent` | boolean |
```
[the remark this answers is not in the document]
Comment (muzea) [reply components/form/index.en-US.md]: I know, but it is still can use

[the remark this answers is not in the document]
Comment (afc163) [reply components/form/index.en-US.md]: Deprecated prop should not be documented.

[the remark this answers is not in the document]
Comment (muzea) [reply components/form/index.en-US.md]: reomved， bug CI looks has some problem,  often hang up recently.

**Comment by afc163** [comment]
Thx! @muzea

**Comment by muzea** [comment]
Close #10976 .

**Pull request merged**